libnmstate/nm/active_connection.py

```diff
@@ -61,20 +66,18 @@ def deactivate(self):
 
     def _safe_deactivate_async(self):
         act_connection = self._nmdev.get_active_connection()
-        mainloop = nmclient.mainloop()
         if not act_connection or act_connection.props.state in (
-            nmclient.NM.ActiveConnectionState.DEACTIVATING,
-            nmclient.NM.ActiveConnectionState.DEACTIVATED,
+            NM.ActiveConnectionState.DEACTIVATING,
+            NM.ActiveConnectionState.DEACTIVATED,
         ):
             # Nothing left to do here, call the next action.
-            mainloop.execute_next_action()
+            self._mainloop.execute_next_action()
             return
 
         user_data = None
-        client = nmclient.client()
-        client.deactivate_connection_async(
+        self._cli.deactivate_connection_async(
```

Comment:
I do not understand why `_cli`, should it be better something like `_client` or `_nmclient`? Thanks.

Reply:
I intend to make internal variable name length shorter.
Not a big deal. If not other objection, I intend to keep that way.

Reply:
my 2 cents: To me "cli" sounds like "command-line interface" not "client" and I prefer more explicit names over names that are ambiguous.

Reply:
All `self._cli` has been changed to `self._client`.